loopd/swapclient_server.go

```diff
@@ -81,6 +81,14 @@ func (s *swapClientServer) LoopOut(ctx context.Context,
 		if err != nil {
 			return nil, fmt.Errorf("decode address: %v", err)
 		}
+
+		// Check that the provided destination address has the
+		// correct format for the active network.
+		if !sweepAddr.IsForNet(s.lnd.ChainParams) {
+			return nil, fmt.Errorf("destination address format " +
```

Comment:
Might be easier to define an `ErrIncorrectChain` error and return it here so that you can match the exact value of the err in a unit test. Can just log the extra details.